src/test/hooks.ts

```diff
@@ -13,7 +13,9 @@ export function attachDefaultHooks () {
   })
 
   afterEach(() => {
-    const pendingMocks = (mock.pendingMocks() as any).filter(item => item !== retrieveServiceTokenMock.interceptors[0]._key)
-    expect(pendingMocks, 'At least one mock were declared but not used').to.be.deep.equal([])
+    if (this && this.currentTest && this.currentTest.state === 'passed') {
```

Comment:
Does `this` happen to be null or undefined? I wonder whether we can get rid of first condition.

Reply:
There were some instances when `this` was null (and seemed to be so correctly) so I added the checks just to be safe.

Reply:
We can switch it to `if (!this || !this.currentTest || this.currentTest.state === 'passed')`